README.md

```diff
@@ -1,4 +1,4 @@
-# Functions Framework
+# Functions Framework [![Documentation](https://img.shields.io/badge/docs-FunctionsFramework-red.svg)](https://googleapis.dev/ruby/google-api-client/latest/Google/Apis.html) [![Gem Version](https://badge.fury.io/rb/functions_framework.svg)](https://badge.fury.io/rb/functions_framework)
```

Comment:
The documentation link should be `https://rubydoc.info/gems/functions_framework/FunctionsFramework`

Reply:
Done.